scripts/dialog.tsx

```diff
@@ -146,7 +163,8 @@ class DialogComponent extends React.Component<any, IDialogComponentState> {
                     <div>{description}</div>
                     <TextBoxComponent value={newTitle} onChange={onTextboxChange} />
                     <ListComponent items={items} onRemove={onRemove} />
-                    <CheckboxComponent checked={openNewWorkItem} onChange={onCheckboxChange} />
+                    <CheckboxComponent checked={openNewWorkItem} onChange={onCheckboxChange} /> <br />
+                    <CheckboxComponent2 checked={copyTags} onChange={onCheckboxChange2} />
                 </div>;
```

Comment:
nit: I would rename this to onCopyTagsChanged. I understand that the previous one said onCheckboxChange, but that is also not correct (we don't have to fix that one as that is old code :))